Contributions/Using_Linux_in_a_Safe_System.md

```diff
@@ -412,6 +412,64 @@ and/or costly.
     of the SoC watchdog toward multiple operating systems, however it
     could go beyond that, and reach higher levels of complexity.
 
+  * **System Decomposition**
+ 
+    The previous points can be leveraged and extended to cases where the
+    safety requirements rely on multiple subsystems, possibly across different
+    software layers.
+    
+    As usual, it is extremely important to assess upfront the requirements
+    for availability.
+
+    A requirement for simply detecting the potential interference can be easier
```

Comment:
@paolonig ASIL Decomposition is an example of applying the principles of system decomposition with a particular set of goals. I believe this document is intended to describe the general approach for system decomposition, rather than ASIL decomposition in particular.

Reply:
@reiterative  fine, I am not suggesting to use the "ASIL Decomposition" wording here, I am just highlighting that we are only focusing on the monitoring of interference failure modes while the monitoring can be used also to detect failure modes deriving from a low systematic capability of the monitored subsystems...

Reply:
I interpret this comment as not requiring changes.